<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListView" name="listView"/>
      <widget class="midie::PianoRollScroll" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="midie::PianoRollWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>80</width>
          <height>518</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="midie::TrackChooser" name="trackComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewSmf"/>
   <addaction name="actionOpenSmf"/>
   <addaction name="actionWriteSmf"/>
   <addaction name="actionRedraw"/>
  </widget>
  <action name="actionOpenSmf">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionWriteSmf">
   <property name="text">
    <string>Write</string>
   </property>
  </action>
  <action name="actionNewSmf">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionRedraw">
   <property name="text">
    <string>Redraw</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>midie::PianoRollWidget</class>
   <extends>QWidget</extends>
   <header>pianorollwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>midie::PianoRollScroll</class>
   <extends>QScrollArea</extends>
   <header>pianorollwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>midie::TrackChooser</class>
   <extends>QComboBox</extends>
   <header>trackchooser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
